### ISSUE TITLE
Remove web preset dependency on @neutrinojs/image-minify

### ISSUE THREAD

**Issue by edmorley** [mozilla-neutrino/neutrino-dev#700]
Since it increases the yarn install time significantly due to the number of compiled packages and yarnpkg/yarn#932. The feature is disabled by default, however this is still a breaking change so would need to wait until Neutrino 9.

See:
https://github.com/mozilla-neutrino/neutrino-dev/pull/673#issuecomment-361320075

**Comment by timkelty** [comment]
At the same time, lets consider if it is worth it to continue the aggregate `@neutrinojs/minify`, or if they should just exist separately.

**Comment by timkelty** [comment]
Related issues and discussion with "default" minification:
https://github.com/mozilla-neutrino/neutrino-dev/issues/678
https://github.com/mozilla-neutrino/neutrino-dev/pull/673

**Comment by ftes** [comment]
Also valuable to avoid compilations on CI servers which may not have required build tools.

E.g. the `mozjpeg` build requires `autoreconf`, and the `pngquant` build requires `libpng-dev`.

```
✖ Error: autoreconf -fiv && ./configure --disable-shared --prefix=".../node_modules/mozjpeg/vendor" ... && make --jobs=8 && make install --jobs=8
/bin/sh: autoreconf: command not found
```
```
✖ Error: pngquant failed to build, make sure that libpng-dev is installed
```

**Comment by edmorley** [comment]
Fixed by #815 which will be in Neutrino 9.
